web/models/build_status_updater.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Bigben.BuildStatusUpdater do
+  alias Bigben.Build
+  alias Bigben.Repo
+
+  def update(build_params) do
+    changeset = Build.changeset(%Build{}, build_params)
+    Repo.insert!(changeset)
+  end
```

Comment:
I think this is simple enough that it could go in the controller. That way you can see at a glance what the controller is doing